packages/ember-runtime/lib/system/core_object.js

```diff
@@ -706,12 +700,55 @@ var ClassMixinProps = {
     @param [arguments]*
     @public
   */
-  create(...args) {
+  create() {
     var C = this;
-    if (args.length > 0) {
-      this._initProperties(args);
+    if (arguments.length === 0) {
+      return new C();
+    } else if (arguments.length === 1) {
+      return new C(arguments[0]);
+    } else if (arguments.length > 1) {
+      let concatenatedProperties = this.concatenatedProperties;
```

Comment:
`this.concatentatedProperties` ` this.mergedProperties` are instance state, not class state. Im not sure how this works?